pages/common/ngrep.md

```diff
@@ -0,0 +1,15 @@
+# ngrep
+
+> Ngrep is a pcap-aware tool that will allow you to specify extended regular or hexadecimal expressions to match against data payloads of packets.
+
+- Monitor all activity crossing source or destination port on network interface dev:
+
+`ngrep -d {{dev}} port {{port}}`
+
+- Monitor all activity crossing source or destination port on any network interface:
```

Comment:
How about placing this example before the previous one? The convention is to go from simpler to more complex/detailed examples.